### DISCUSSION TITLE
Exclude files by path AND size

### DISCUSSION THREAD

**Discussion by Kibubu** [abraunegg/onedrive#2131] [Q&A] [answered]
Currently i'm using the sync_list to exclude specific directories but I would actually would like to only exclude specific file-types or file-sizes in said directory. Is there a way to provide logic rules to sync_list? I could possibly create a new configuration just for that folder but maybe there is a better way.

```
.
├── file1.csv < 100 MB
├── file2.csv < 100 MB
└── Media
    ├── Project1
    |   ├── file.txt < 100 MB
    │   └── file.jpg < 100 MB
    ├── Project2
    |   ├── file.txt < 100 MB
    │   └── file.avi > 100 MB
    └── Project3
        ├── file1.jpg < 100 MB
        └── raw
            ├── file1.avi > 100 MB
            ├── file2.avi > 100 MB
            └── file3.avi > 100 MB

```

I would like to sync all files in Media that are smaller that 100 MB

**Comment by abraunegg** [comment, marked as answer]
@Kibubu 
You can use the following:
* Use `--skip-size ARG` as part of a CLI command to skip new files larger than this size (in MB)
* Use `skip_size = "value"` as part of your 'config' file

This will be a global option, so will affect all directories, not just your 'Media' folder.



Documentation is also updated to reference: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#how-to-skip-files-larger-than-a-certain-size-from-syncing

**Reply by Kibubu** [reply]
Thanks! I can work with that.

I will then start two daemons and organize my folders as follows.

```
. (managed by damon1, /Media excluded)
├── file1.csv < 100 MB
├── file2.csv < 100 MB
└── Media (managed by daemon2, with skip-size)
    ├── Project1
    |   ├── file.txt < 100 MB
    │   └── file.jpg < 100 MB
    ├── Project2
    |   ├── file.txt < 100 MB
    │   └── file.avi > 100 MB
    └── Project3
        ├── file1.jpg < 100 MB
        └── raw
            ├── file1.avi > 100 MB
            ├── file2.avi > 100 MB
            └── file3.avi > 100 MB
```
